ops/op2/dispatch_fast.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+use super::generator_state::GeneratorState;
+use super::signature::Arg;
+use super::signature::NumericArg;
+use super::signature::ParsedSignature;
+use super::signature::RetVal;
+use super::V8MappingError;
+use proc_macro2::TokenStream;
+use quote::format_ident;
+use quote::quote;
+
+#[allow(unused)]
+#[derive(Debug, Default, PartialEq, Clone)]
+pub(crate) enum FastValue {
+  #[default]
+  Void,
+  Bool,
+  U32,
+  I32,
+  U64,
+  I64,
+  F32,
+  F64,
+  Pointer,
+  V8Value,
+  Uint8Array,
+  Uint32Array,
+  Float64Array,
+  SeqOneByteString,
+}
+
+impl FastValue {
+  /// Quote fast value type.
+  fn quote_rust_type(&self) -> TokenStream {
+    match self {
+      FastValue::Void => quote!(()),
+      FastValue::Bool => quote!(bool),
+      FastValue::U32 => quote!(u32),
+      FastValue::I32 => quote!(i32),
+      FastValue::U64 => quote!(u64),
+      FastValue::I64 => quote!(i64),
+      FastValue::F32 => quote!(f32),
+      FastValue::F64 => quote!(f64),
+      FastValue::Pointer => quote!(*mut ::std::ffi::c_void),
+      FastValue::V8Value => quote!(v8::Local<v8::Value>),
```

Comment:
Maybe this one should use the same strategy as https://github.com/denoland/deno/pull/19593?

Reply:
In this case we should be probably want to import `v8` to avoid having to pass the prefix up, but yeah this wouldn't work. I'll change to unimplemented for now.